src/mesido/electricity_physics_mixin.py

```diff
@@ -17,6 +18,28 @@
 logger = logging.getLogger("mesido")
 
 
+class ElectrolyzerOption(IntEnum):
+    """
+    Enumeration for the possible options to model the electrolyzer.
+    For detailed calculations related to these options, see the equations themselves.
+
+    CONSTANT_EFFICIENCY
+        A constant efficiency is used to determine the H2 produced per energy electricity entering
+
+    LINEARIZED_INEQUALITY
+        The efficiency curve is linearized in 3 lines, which are all inequalities and with proper
+        goals will move towards these lines
+
+    PIECE_WISE_LINEAR_EQUALITY
+        The efficiency curve is linearized in 3 lines, which are all equalities using binary
+        variables and the big-M method to select the relevant lines.
+    """
+
+    CONSTANT_EFFICIENCY = 1
+    LINEARIZED_INEQUALITY = 2
+    PIECE_WISE_LINEAR_EQUALITY = 3
```

Comment:
I would prefer if we standardise and change these names to be similar to head_loss, see examples below:
* HeadLossOption.LINEARIZED_ONE_LINE_EQUALITY : the word "ONE" is included so that you know that the linear line quantity is not optional, but rather fixed
* HeadLossOption.LINEARIZED_N_LINES_WEAK_INEQUALITY: the character "_N_" indicates that the number of linear lines are adjustable and there are "N" number of them.

So in this case if the 3 lines are fixed (not adjustable) I would use "_THREE_" in the variable name, else if adjustable (with potentially a default value of 3) then I would add the character "_N_", similar to what was done in the head loss options

Reply:
I can update those. But I'll use, THREE in the name as it is fixed to this for a reason, due to the curve of the electrolyzer efficiency.